examples/plain-javascript/README.md

```diff
@@ -22,7 +22,7 @@ you want. But, as an exercise of this demo, we'll download it ourselves. So:
 1. Click at [cldrjs releases tab](https://github.com/rxaviers/cldrjs/releases).
 1. Download the latest package.
 1. Unzip it.
-1. Move its `dist/` files into `cldrjs` of this directory.
+1. Create a `cldrjs` directory alongside `index.html` and `README.md` and move the cldrjs `dist/` files into it.
```

Comment:
Looks fine to me, @rxaviers . One thing that _might_ make it more clear would be to break it up into 2 steps:

... 
1. Inside the unzipped directory, rename the `dist` directory to  `cldrjs`.
1. Move the renamed `cldrjs` directory from the unzipped folder to this demo's directory so that it appears alongside `index.html` and `README.md`.

I don't know, though. Maybe that just makes it overly complicated.